src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -228,5 +229,17 @@ public static class Task
                 public static readonly string DisplayName = "task.displayname";
             }
         }
+
+        public class Release
+        {
+            public class Path
+            {
+                public static readonly string WorkingDirectory = "r";
```

Comment:
I am not introducing a new concept. Looks like the variable name that i have is misleading. (I am bad at naming variables, methods). 
I am infact modifying the Agent.ReleaseDirectory variable.
Earlier it was _work/{SmallHash}
Now, _work/r/1/a

Instead of hash now i have is /r/1/a . I ll make sure that i dont use the word workingDirectory. I understand that it adds confusion.